test/exec/test_any_sender.cpp

```diff
@@ -361,19 +366,19 @@ struct stopped_receiver : stopped_receiver_base {
 static_assert(
   receiver_of<stopped_receiver, completion_signatures<set_value_t(int), set_stopped_t()>>);
 
-TEST_CASE("any_sender does not connect with stop token", "[types][any_sender]") {
-  using unstoppable_sender = any_sender_of<set_value_t(int), set_stopped_t()>;
-  unstoppable_sender sender = when_any(just(21));
+TEST_CASE("any_sender - does connect with stop token", "[types][any_sender]") {
+  using stoppable_sender = any_sender_of<set_value_t(int), set_stopped_t()>;
+  stoppable_sender sender = when_any(just(21));
   in_place_stop_source stop_source{};
-  stopped_receiver receiver{stop_source.get_token(), false};
+  stopped_receiver receiver{stop_source.get_token(), true};
```

Comment:
This stoppable_sender should be able to type-erase a sender with any kind of stop token, right? Not just in_place and never.

Reply:
Thank you. Other than that, this looks great.